tests/stub/transport/test_handshakes.py

```diff
@@ -53,12 +53,12 @@ def _run(self, handshake, handshake_response):
             vars_["#HANDSHAKE_RESPONSE#"] = handshake_response
         with self._get_session(script_path, vars_=vars_) as session:
             session.run("RETURN 1 AS n")
```

Comment:
```suggestion
            session.run("RETURN 1 AS n").consume()
```

Testing changes on JS side requires the result to be consumed, otherwise the server is shut down mid-work.

There might be a better fix. But this makes the tests work.